ContactManager/src/com/contacts/manager/ContactManager.java

```diff
@@ -0,0 +1,77 @@
+package com.contacts.manager;
+
+import com.contacts.model.Contact;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class ContactManager {
+
+    private static final ContactManager myClass = new ContactManager();
+    private static List<Contact> contacts = new ArrayList<Contact>();
+
+    private ContactManager() {
+
+    }
+
+    public static ContactManager getInstance() {
+        return myClass;
+    }
+
+    public void addContacts(Contact value) {
+        contacts.add(value);
+        System.out.println("    [ CONTACT SAVED ]");
```

Comment:
@cesar1414 You can refactor this code and separate the console interaction in another class.